translations/es.ts

```diff
@@ -1988,13 +1988,13 @@ Por favor usa otra.</translation>
     <message>
         <location filename="../src/widget/form/profileform.cpp" line="358"/>
         <source>Please enter a new password.</source>
-        <translation>Ingresa tu nueva contraseña.</translation>
+        <translation>Por favor, ingresa tu nueva contraseña.</translation>
```

Comment:
pensé que así es menos tono de mando....
removing the please from Spanish version again...